workspaces/cli-server/src/config.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  createCliConfig,
+  Env,
+  CliConfigObject,
+  isEnvTrue,
+  PackageJson,
+} from '@useoptic/cli-shared';
+const pkg = require('../package.json') as PackageJson;
```

Comment:
I have a vague memory of issues using the package json in the past but if typescript doesn't complain and the package json is present in the published package then it should work.

Reply:
Typescript complains if you use `import`, but require is just another function-call, so fine. Safer, if we didn't control the `package.json` would be to use [`read-package-json`](https://www.npmjs.com/package/read-package-json) which performs some validation steps, but since these packages won't even publish without a valid `package.json`, I didn't deem the necessary at this point.